### ISSUE TITLE
for those having issues regarding motion and orientation for IOS 13+

### ISSUE THREAD

**Issue by micahnut** [aframevr/aframe#4277]
i have recently updated my iphone. I found this article to be helpful. my a-frame project worked the way it worked before updating to IOS 13. 

https://medium.com/@leemartin/how-to-request-device-motion-and-orientation-permission-in-ios-13-74fc9d6cd140

<img width="732" alt="Screen Shot 2019-09-24 at 12 49 10 PM" src="https://user-images.githubusercontent.com/22316534/65481900-bfd96900-dec9-11e9-922f-3700982957b0.png">

**Comment by dmarcos** [comment]
It seems it needs to be triggered by user gesture so it won’t work on load as before, right?

**Comment by timothyallan** [comment]
Yep, it needs a user gesture for a "granted" response.

**Comment by ffmaer** [comment]
Following https://medium.com/@leemartin/how-to-request-device-motion-and-orientation-permission-in-ios-13-74fc9d6cd140, I got errors that says the following:

NotAllowedError: Requesting device orientation or motion access requires a user gesture to prompt

I clicked the touch-screen. I used a finger to swipe the touch-screen. Both of these actions did not cause permission requesting prompt to show up.

What is a user gesture?
